README.md

```diff
@@ -136,6 +136,34 @@ compile "com.android.support:appcompat-v7:$BAR"
 compile "com.android.support:palette-v7:$BAR"
 ```
 
+NOTE :
+The support libraries are now available through Google's Maven repository, so ensure that you have that added to your project's repositories.
+
+Open the `build.gradle` file for your project and modify it according to your gradle plugin version.
+
+If your gradle plugin is above 2.3.3 
```

Comment:
Let's just drop this section.  Simpler to add the long form, which will work for everyone,